linkingFuncs.py

```diff
@@ -25,6 +25,62 @@ def get_id(username, cur):
         return None
     return fetch[0]
 
+@app.route("/buy/<username>/<tik>/<volume>")
+def buy(username, tik, volume):
+    # establish connection
+    cnx = mysql.connector.connect(user='root', password='Valentino46', database='StonkLabs')
+    # create a cursor 
+    cur = cnx.cursor(buffered=True)
+    # get current price
+    price =  search_tiker(tik).get('price')
+    if(price == {}):
+        return { 'Action': False}
+    id = get_id(username, cur)
+    if (id ==  None):
+        return { 'Action': False}
+    time = datetime.datetime.now()
+    cur.execute("INSERT INTO Transactions Values ('%d', '%s', '%d', 'BUY', '%f', '%s', 0)" % (
+        id, tik, volume, price, time))
+    # Commit change 
+    cnx.commit()
+    # Close connections 
+    cnx.close()
+    return { 'Action': True}
+
+@app.route("/sell/<username>/<tik>/<volume>")
+def sell(username, tik, volume):
+        # establish connection
+    cnx = mysql.connector.connect(user='root', password='Valentino46', database='StonkLabs')
+    # create a cursor 
+    cur = cnx.cursor(buffered=True)
+    # get current price
+    price =  search_tiker(tik).get('price')
+    if(price == {}):
+        return { 'Action': False}
+    id = get_id(username, cur)
+    if (id ==  None):
+        return { 'Action': False}   
+    time = datetime.datetime.now()
```

Comment:
Date and time may not need to be implemented

Reply:
The time is not being used at the moment. Will look into removing this column from the database.